docs/source/gbd2019_models/risk_effects/zinc_deficiency/index.rst

```diff
@@ -0,0 +1,254 @@
+.. _2019_risk_effect_zinc_deficiency:
+
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1
+  ---------------
+
+  Section Level 2
+  +++++++++++++++
+
+  Section Level 3
+  ^^^^^^^^^^^^^^^
+
+  Section Level 4
+  ~~~~~~~~~~~~~~~
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+============================
+Zinc Deficiency Risk Effects
+============================
+
+.. contents::
+   :local:
+   :depth: 2
+
+Risk Overview
+-------------
+
+.. todo::
+
+	Provide a brief description of the risk, including potential opportunities for confounding (factors that may cause or be associated with the risk exposure), effect modification/generalizability, etc. by any relevant variables. Note that literature reviews and speaking with the GBD risk modeler will be good resources for this.
+
+GBD 2019 Modeling Strategy
+--------------------------
+
+.. note::
+
+	This section will describe the GBD modeling strategy for risk effects. For a description of GBD modeling strategy for risk exposure, see the :ref:`risk exposure <2019_risk_exposure_zinc_deficiency>` page.
+
+.. todo::
+
+	Provide a brief overview of how the risk affects different outcomes, including data sources used by GBD, GBD assumptions, etc. Note that the [GBD-2019-Risk-Factors-Appendix-Risk-Effects-Model-Template]_ is a good source for this information in addition to the GBD risk modeler.
```

Comment:
This citation link is not working when I build the html. Probably because the citation still says GBD 2017...